docs/quickstarts/ratify-high-availability.md

```diff
@@ -2,7 +2,36 @@
 
 The default Ratify installation relies on a single Ratify pod processing all requests. For higher performance and availability requirements, Ratify can be set to run with multiple replicas and a shared state store.
 
-## Add Helm Chart Dependencies
+Ratify installation/upgrade for HA scenarios can be done via a `helmfile` or manual installation steps. Both options are outlined in this document.
+
+## Automated Installation
+**Prerequisites**
+```bash
+# Download and install yq
+	curl -L https://github.com/mikefarah/yq/releases/download/v4.34.2/yq_linux_amd64 --output /usr/bin/yq && chmod +x /usr/bin/yq
+```
+```bash
+# Download and install helmfile
+  curl -LO https://github.com/helmfile/helmfile/releases/download/v0.155.0/helmfile_0.155.0_linux_amd64.tar.gz
+  mkdir helmfile-install
+  tar -zxf helmfile*.tar.gz -C helmfile-install/
+  mv helmfile-install/helmfile /usr/bin
+  rm -rf helmfile*.tar.gz helmfile-install/
+```
+> Note: Helmfile does not have a stable release and thus is not recommended for production environements. 
```

Comment:
Suggestion to move notes to the top, before the install command. Looks like there is a typo here , "environements", environments?